internal/k8s/ako_init.go

```diff
@@ -554,6 +562,21 @@ func (c *AviController) DeleteModels() {
 		utils.AviLog.Infof("Deleting objects for model: %s", modelName)
 		sharedQueue.Workqueue[bkt].AddRateLimited(modelName)
 	}
+
+	// Wait for maximum 30 minutes for the sync to get completed
+	timeout := make(chan bool, 1)
+	go func() {
+		time.Sleep(lib.AviObjDeletionTime * time.Minute)
+		timeout <- true
+	}()
+	lib.SetConfigDeleteSyncChan()
+	select {
+	case <-lib.ConfigDeleteSyncChan:
+		utils.AviLog.Infof("Processing done for deleteConfig, user would be notified through statefulset update")
+	case <-timeout:
+		utils.AviLog.Warnf("Timed out while waiting for rest layer to respond for delete config")
+	}
+	status.AddStatefulSetStatus(lib.ObjectDeletionDoneStatus)
```

Comment:
Can we pls check for the timeout here and then update the status as `objDeletionTimeout`?